p256/src/arithmetic/field.rs

```diff
@@ -201,12 +202,20 @@ impl FieldElement {
 
     /// Returns self - rhs mod p
     pub const fn subtract(&self, rhs: &Self) -> Self {
+        let (_, result) = Self::sub_inner(
+            self.0[0], self.0[1], self.0[2], self.0[3], 0, rhs.0[0], rhs.0[1], rhs.0[2], rhs.0[3],
+            0,
+        );
+        result
+    }
+
+    /// Returns self - rhs mod p
+    pub const fn informed_subtract(&self, rhs: &Self) -> (u64, Self) {
```

Comment:
FWIW, our convention for this is generally to make the borrow the second element in the tuple. See `sbb`, for example.

I'm also not sure about the name here, or making it part of the public API.

Hopefully a lot of this code will be cleaner as we transition to `crypto_bigint::UInt`s for the inner type for field elements.

Reply:
> FWIW, our convention for this is generally to make the borrow the second element in the tuple. See `sbb`, for example.

Makes sens, fixed

> 
> I'm also not sure about the name here, or making it part of the public API.

Yeah.. agreed, I've limited it to crate only

> 
> Hopefully a lot of this code will be cleaner as we transition to `crypto_bigint::UInt`s for the inner type for field elements.

For sure, yeah. This appears to work in the mean-time and gets of of the wild patch train we had to keep up with